src/views/Tenant/TenantView.js

```diff
@@ -0,0 +1,264 @@
+import React, { useEffect, useState } from "react";
+import * as Yup from "yup";
+import { useParams } from "react-router-dom";
+import * as axios from "axios";
+import { SearchPanel, SearchPanelVariant } from "react-search-panel";
+import ToggleEditTable from "../../components/ToggleEditTable";
+
+// Configure validation schema for edit form
+const validationSchema = Yup.object().shape({
+  firstName: Yup.string()
+    .max(255, "Must be shorter than 255 Characters")
+    .required("Must enter a First Name"),
+  lastName: Yup.string()
+    .max(255, "Must be shorter than 255 Characters")
+    .required("Must enter a Last Name"),
+  phone: Yup.string()
+    .min(
+      5,
+      "*Number must contain at least 5 digits to be a valid phone/text number",
+    )
+    .max(20, "*Numbers can't be longer than 20 digits")
+    .required("*a valid phone number is required"),
+});
+
+const Tenant = () => {
+  // Get input tenant id
+  const { id } = useParams();
+
+  const initialState = {
+    tenant: null,
+    property: null,
+    tickets: null,
+  };
+
+  const [state, setState] = useState(initialState);
+  const [isEditing, setEditingStatus] = useState(false);
+  const [staffSearchText, setStaffSearchText] = useState("");
+  const [staffSearchResults, setStaffSearchResults] = useState([]);
+  const [staffSelections, setStaffSelections] = useState(null);
+
+  const tabs = [
+    { id: "Ongoing", label: "Ongoing" },
+    { id: "Closed", label: "Closed" },
+  ];
+  const [activeTab, setActiveTab] = useState(tabs[0].id);
+
+  /**
+   * Handle activating edit form
+   */
+  const handleEditToggle = () => setEditingStatus(!isEditing);
+  const onFormikSubmit = (values, { setSubmitting }) => {
+    setSubmitting(true);
```

Comment:
Do we have an issue created to have the form actually save to the backend? If we don't do it with this PR

Reply:
I saw you added #304, so I'll work on that next. Thanks!